### PR TITLE
Updates to change detection algorithms

### PR DESCRIPTION
**Pull request by bmeyers**
Recent changes to our model of smooth, periodic functions led to issues with using QSS in nonconvex mode for change point detection. We are deprecating the use of QSS and standardizing the use of CLARABEL to solve the SD problems. In addition, we are no longer using the non-convex formulation for change point detection. Instead, we solve the L1-heuristic problem and polish the solution.

In this PR, we make changes to the two subroutines that use this pattern: time shift detection and capacity change detection. Also, we update the unit test fixtures to account for the new SD models, after manually checking the outputs in the jupyter notebooks which are in the test fixtures.

### CODE REVIEW

**Comment by bmeyers** [comment]
@pluflou help, conda is failing again! 😅

File: solardatatools/_osd_signal_decompositions.py [hunk 1]
```diff
@@ -85,33 +87,35 @@ def _osd_l2_l1d1_l2d2p365(
...
-        w1 /= 1e4
-        w2 /= 1e4
-
     c1 = SumSquare(weight=w0)
     T = len(signal)
-    c2 = Fourier(3, T, 365.2425, weight=1e-3)
-
-    if sum_card:
-        c3 = SumCard(weight=w1, diff=1)
+    c2 = Aggregate([Fourier(3, T, 365.2425, weight=1e-3), AverageEqual(0)])
```
Comment (bmeyers) [inline solardatatools/_osd_signal_decompositions.py:92]: pass w2

File: solardatatools/data_handler.py [hunk 2]
```diff
@@ -184,14 +184,14 @@ def run_pipeline(
         start_day_ix=None,
         end_day_ix=None,
         w1=None,
-        w2=1e5,
+        w2=1e-3,
```
Comment (bmeyers) [inline solardatatools/data_handler.py:187]: remove

**Review by Thistleman — APPROVED** [review]
The code review looked good and the explanations made sense

**Review by pluflou — APPROVED** [review]

**Pull request merged**